<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>defaultlook</class>
 <widget class="QMainWindow" name="defaultlook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MX Default Look</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-desktop-theme">
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Panel Options</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Theme Options</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkHorz">
      <property name="text">
       <string>Display panel horizontally</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkLightTheme">
      <property name="text">
       <string>Default light theme</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkVert">
      <property name="text">
       <string>Display panel vertically</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkDarkTheme">
      <property name="text">
       <string>Default dark theme</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkFirefox">
      <property name="toolTip">
       <string>Fix some website fields when using dark theme</string>
      </property>
      <property name="text">
       <string>Firefox dark theme tweak</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <spacer name="horizontalSpacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>29</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelMX">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../../../../../../../usr/share/mx-panel-orientation/icons/mxfcelogo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>29</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="buttonApply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../../../usr/share/mx-panel-orientation/icons/dialog-ok.png</normaloff>../../../../../../../../usr/share/mx-panel-orientation/icons/dialog-ok.png</iconset>
        </property>
        <property name="shortcut">
         <string/>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="buttonCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Quit application</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../../../usr/share/mx-panel-orientation/icons/gtk-cancel.png</normaloff>../../../../../../../../usr/share/mx-panel-orientation/icons/gtk-cancel.png</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+N</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonAbout">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../usr/share/mx-defaultlook/icons/accessories-text-editor.png</normaloff>../../../usr/share/mx-defaultlook/icons/accessories-text-editor.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonHelp">
        <property name="text">
         <string>Help</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../usr/share/mx-defaultlook/icons/gnome-help.png</normaloff>../../../usr/share/mx-defaultlook/icons/gnome-help.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="checkHexchat">
      <property name="text">
       <string>Fix HexChat input box in dark theme</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
